src/mock/compilation.rs

```diff
@@ -11,21 +11,20 @@ pub fn compile_configuration(configuration_content: &str) -> Result<Vec<Expectat
 mod test {
     use super::*;
     use crate::mock::model::{Expectation, HttpMethod, HttpRequest, HttpResponse};
+    use std::collections::BTreeMap;
```

Comment:
Why could we not use a standard HashMap ?

Reply:
We could, the memory footprint will be higher

Reply:
But you're right hasmap is a better choice, I will change that tomorrow

Reply:
done